source/jormungandr/jormungandr/scenarios/helper_classes/helper_utils.py

```diff
@@ -126,11 +126,11 @@ def _extend_pt_sections_with_direct_path(pt_journey, dp_journey):
         pt_journey.sections.extend(dp_journey.journeys[0].sections)
 
 
-def _extend_journey(pt_journey, fallback_dp, fallback_extremity, is_start_fallback):
+def _extend_journey(pt_journey, fallback_dp, fallback_period_extremity):
     """
-    :param fallback_extremity: is a PeriodExtremity (a datetime and it's meaning on the fallback period)
+    :param fallback_period_extremity: is a PeriodExtremity (a datetime and it's meaning on the fallback period)
```

Comment:
*its

Reply:
the way I initially meant it, "fallback_period_extremity" is the subject of the phrase, so it's fine to me.